sdk/src/sysvar/recent_blockhashes.rs

```diff
@@ -74,6 +75,13 @@ where
     account
 }
 
+pub fn create_test_recent_blockhashes(start: usize) -> RecentBlockhashes {
```

Comment:
can you move this down in to `mod tests {` ?

Reply:
I'll try and see if Rust can still find it from other packages

Reply:
It will only be able to find them if you put it in `mod tests` without a `#[cfg(test)]`.  Traditionally we haven't done that, but could start.

Reply:
Ah! That's it.  I knew I'd fought this fight before.  Same deal for tagging the function with `#[cfg(test)]` ?

Reply:
indeed